src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/ClientConfigurationSettings.java

```diff
@@ -137,11 +161,36 @@ public static class ClientConfigurationWorkspaceSymbolSettings {
      */
     public boolean caseSensitive = false;
 
+    /**
+     * The line comment prefix in the language grammar. Defaults to none.
+     */
+    public @Nullable String lineCommentPrefix = null;
```

Comment:
And these are the language grammar-level comment prefix/suffixes and statement terminator characters for the language (if it even supports statement terminators).